trojsten/contests/tests.py

```diff
@@ -524,3 +526,93 @@ def setUp(self):
 
     def test_latest_finished_for_competition(self):
         self.assertEqual(Round.objects.latest_finished_for_competition(self.competition).number, 2)
+
+class DashboardTest(TestCase):
+    def setUp(self):
+        self.site = Site.objects.get(pk=settings.SITE_ID)
+        self.url = reverse('dashboard')
+
+    def test_dashboard_no_round(self):
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, _("There is no active round currently."))
+
+    def test_active_round(self):
+        competition = Competition.objects.create(name='TestCompetition')
+        competition.sites.add(self.site)
+        semester = Semester.objects.create(
+            number=1, name='Test semester 1', competition=competition, year=1
+        )
+        round_ = Round.objects.create(number=1, semester=semester, solutions_visible=True, visible=True)
```

Comment:
Nit: round

Viem ze sa to bije s built-in funkciou, ale tu to nevadi.
Ked uz tak _round